build/steps.yml

```diff
@@ -1,35 +1,37 @@
 ##
 # Build and test IQ#.
 ##
+parameters:
```

Comment:
It feels we need different steps file for conda. They can all publish to the same artifact.

Reply:
Sounds good, happy to remove that from pack.ps1, then. That's definitely a lot simpler than trying to make pack.ps1 do two different things.